web/admin/application/library/IvozProvider/Klear/Filter/Extensions.php

```diff
@@ -11,19 +11,18 @@ class IvozProvider_Klear_Filter_Extensions extends IvozProvider_Klear_Filter_Com
 
     public function setRouteDispatcher(KlearMatrix_Model_RouteDispatcher $routeDispatcher)
     {
+        $controller = $routeDispatcher->getControllerName();
+        if (!in_array($controller, ['edit', 'new'])) {
```

Comment:
in_array should always be in strict mode: !in_array($controller, ['edit', 'new'], true)